src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -40,6 +40,71 @@ def test_get_error(session):
         assert e.description.find('Attribute is read-only.') != -1
 
 
+def test_commit_after_initiate(session):
+    try:
+        with session.initiate():
+            session.commit()
+    except nidcpower.Error as e:
+        assert False
+
+
+def test_get_self_cal_last_date_and_time(session):
```

Comment:
These tests are unfortunate because they don't really tell us much... for example, the interesting aspect of session.get_self_cal_last_date_and_time() is that it returns several values - yet that's not being tested.

I don't have a great suggestion on how to improve this, however. But it is a definite hole in our coverage.

Reply:
Probably we can add these kind of tests to another python file which will be run with our internal testsuites. 

Thoughts ?

Reply:
I don't think it's worth the complexity of having different tests in NI's internal test suites (which have hardware) and the stuff we run on nimi-bot (CI system tests).

This might simply end up being a gap in our test coverage.